typespec-extension/src/main/java/com/azure/autorest/TypeSpecPlugin.java

```diff
@@ -122,8 +124,7 @@ public String handlePartialUpdate(String filePath, String generatedContent) {
         SETTINGS_MAP.put("generate-sync-async-clients", true);
         SETTINGS_MAP.put("generate-builder-per-client", false);
         SETTINGS_MAP.put("sync-methods", "all");
-        // TODO(xiaofei) set to true when PagedIterable::mapPage is fixed in azure-core
-        SETTINGS_MAP.put("enable-sync-stack", false);
+        SETTINGS_MAP.put("enable-sync-stack", true);
```

Comment:
OK with it, but I remember the fix is merged but not released in 1.40 core.

Reply:
Yeah, we will release azure-core this week. So, we should be able to make this the default and I will merge this PR after the next core release.